helm/deploy.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+# ModelExpress Helm Chart Deployment Script
+# This script helps deploy ModelExpress using Helm
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+# Default values
+RELEASE_NAME="model-express"
+NAMESPACE="model-express"
+VALUES_FILE=""
+DRY_RUN=false
+UPGRADE=false
+
+# Function to print colored output
+print_status() {
+    echo -e "${BLUE}[INFO]${NC} $1"
+}
+
+print_success() {
+    echo -e "${GREEN}[SUCCESS]${NC} $1"
+}
+
+print_warning() {
+    echo -e "${YELLOW}[WARNING]${NC} $1"
+}
+
+print_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+# Function to show usage
+show_usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Options:
+    -r, --release-name NAME    Release name (default: model-express)
+    -n, --namespace NAME       Kubernetes namespace (default: model-express)
+    -f, --values FILE          Values file to use (e.g., values-production.yaml)
+    -d, --dry-run              Perform a dry run
+    -u, --upgrade              Upgrade existing release
+    -h, --help                 Show this help message
+
+Examples:
+    # Deploy with default values
+    $0
+
+    # Deploy with production values
+    $0 -f values-production.yaml
+
+    # Deploy with custom release name and namespace
+    $0 -r my-model-express -n my-namespace
+
+    # Perform a dry run
+    $0 -d
+
+    # Upgrade existing release
+    $0 -u -f values-production.yaml
+
+EOF
+}
+
+# Function to check prerequisites
+check_prerequisites() {
+    print_status "Checking prerequisites..."
```

Comment:
Kind of a nit, but this isn't verifying if the deployment files are there. This could theoretically be used in conjunction of `$(dirname $0)` to find the default files there.